makefiles/pseudomodules.inc.mk

```diff
@@ -307,7 +307,14 @@ PSEUDOMODULES += md5sum
 ## @}
 PSEUDOMODULES += mtd_write_page
 PSEUDOMODULES += nanocoap_%
+PSEUDOMODULES += netif_dev_default
+PSEUDOMODULES += netif_dev_ieee802154_default
+## @defgroup net_netdev_default  netdev_default
+## @ingroup net_netif
+## @{
+## @deprecated  Use module `netif_dev_default` instead; will be removed after 2023.07 release.
 PSEUDOMODULES += netdev_default
```

Comment:
If you deprecate that module you should also update all the applications to no longer use the deprecated module.

    find -name Makefile -exec sed -i s/netdev_default/netif_dev_default/g {} \;

Reply:
done!

Reply:
urg, now CI is unhappy :(

Reply:
I just realized not all occurrences where updated... it should work now